### DISCUSSION TITLE
@entangle causes error : Uncaught SyntaxError: illegal character U+0040

### DISCUSSION THREAD

**Discussion by Christopher-Law** [alpinejs/alpine#3389] [7. Bugs (archived)]
When attempting to use Livewire/Alpine/Jetstream with the @entangle directive, I get the following error: 
`Uncaught SyntaxError: illegal character U+0040`

This character code is a '@' 

`<x-jet-form-section x-data="{ biography : @entangle('biography') }">`

` <x-jet-textarea  x-text="biography" />`

I installed Alpine via NPM. 
Browser: Firefox

**Comment by martyf** [comment]
Did you figure this one out? 

I've just tried to use `@entangle` and get the same issue, Livewire/Alpine on a data-section of a Statamic site.
